integrations/block-data-api.php

```diff
@@ -30,11 +38,6 @@ class BlockDataApiIntegration extends Integration {
 	public function load(): void {
 		// Wait until plugins_loaded to give precedence to the plugin in the customer repo.
 		add_action( 'plugins_loaded', function() {
-			// Do not load plugin if already loaded by customer code.
```

Comment:
After this refactoring now our integrations must provide `is_loaded` function and based on this we will call the `load` function and there is also a [test](https://github.com/Automattic/vip-go-mu-plugins/blob/e8c247c88c604fb36ca47228a423f97b9cfdb1cb/tests/integrations/test-integration.php#L40C2-L54C3) present as safeguard.

So I think keeping current implementation is better because its cleaner.

Reply:
See new comment with proposed fix.